Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -271,6 +364,14 @@ public ManipulationEvent OnHoverExited
         private TwoHandScaleLogic scaleLogic;
         private TwoHandRotateLogic rotateLogic;
 
+        private ElasticSystem<Vector3> translationElastic;
+        private ElasticSystem<Quaternion> rotationElastic;
+        private ElasticSystem<Vector3> scaleElastic;
+
+        // Magnitude of the velocity at which the elastic systems will
+        // cease being simulated (if enabled) and the object will stop updating/moving.
+        private readonly static float elasticVelocityThreshold = 0.001f;
```

Comment:
Nit: I think for these cases const might be more appropriate:

https://lassiautio.com/2018/01/21/constant-or-static-readonly/

Reply:
Now using `const`.